CLI/model_loader.py

```
@@ -35,3 +35,38 @@ def load_t5_headline_gen():
+            created_at DATETIME DEFAULT CURRENT_TIMESTAMP
+        )
+    """)
+
+    sql = """
+        INSERT INTO news_summary (category, url, title_summary, body_summary, article_date)
+        VALUES (%s, %s, %s, %s, %s)
+    """
+    cursor.execute(sql, (category, url, title_summary, body_summary, article_date))
+
+    cursor.close()
+    conn.close()
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)